examples/02-plot/backface_props.py

```diff
@@ -0,0 +1,84 @@
+"""
+.. _backface_props_example:
+
+Setting Backface Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default front and backface rendering uses the same properties. In certain
+situations it can be useful to set different properties for backfaces than
+for frontfaces.
+
+One straightforward example is when a closed (or close enough) surface has a
+different color on the inside. Note that the notion of "inside" and "outside"
+depend on the orientation of the surface normals:
```

Comment:
```suggestion
By default front and backface rendering uses the same properties. In certain
situations it can be useful to set different properties for backfaces than
for frontfaces.

One straightforward example is when a closed (or close enough) surface has a
different color on the inside. Note that the notion of "inside" and "outside"
depend on the orientation of the surface normals:
```